### DISCUSSION TITLE
Repositório Renomeado

### DISCUSSION THREAD

**Discussion by pedroleone** [pedroleone/osr-srd#12] [Announcements]
Recentemente havia reparado que o nome do repositório estava incorreto, como osr-srd ao invés de ose-srd. Acabei de renomear o repositório para o nome correto. Isso não deve trazer nenhum problema, pois o Github redireciona qualquer ação no endereço antigo para o novo, mas de qualquer forma fiquem atentos.

**Comment by nerun** [comment]
Eu reparei, mas achei que fosse por causa do movimento `Old School Renaissance`. 😂
